sky/shell/platform/android/org/domokit/sky/shell/SkyApplication.java

```diff
@@ -82,6 +84,14 @@ public void connectToService(Context context, Core core, MessagePipeHandle pipe)
             }
         });
 
+        registry.register(GcmService.MANAGER.getName(), new ServiceFactory() {
```

Comment:
Fitness.apk should then use an application class that subclasses SkyApplication and override onServiceRegistryAvailable to register this service.